### PR TITLE
Orca: enable openvino estimator test

### PR DESCRIPTION
**Pull request by hamham223**
### 1. Why the change?

Previously, the python script [test_estimator_openvino.py](https://github.com/intel-analytics/BigDL/blob/main/python/orca/test/bigdl/orca/learn/spark/test_estimator_openvino.py) is not tested. This PR tries to enable the test.

### 2. Summary of the change

+ install dependencies in [.github/actions/orca/orca-python-horovod-tf2-py37-spark3-action/pr-test/action.yml](https://github.com/intel-analytics/BigDL/pull/7431/files#diff-de8fe6f132a4eeeb2756955cd1942ecfef493479043ff7db9ed1b1a95cfbc194)
+ add test in [python/orca/dev/test/run-pytests-horovod-tf.sh](https://github.com/intel-analytics/BigDL/pull/7431/files#diff-8f398140218546939b8542a2c2d425003d1b1ac40c2ca6ea9994d124fb40a188)

### 3. How to test?

unit test:
- [x] [Orca-Python-Py37-Spark3-Openvino](https://github.com/intel-analytics/BigDL/actions/runs/4131937925/jobs/7140387947)

### 4. New dependencies

- [x] Openvino Package Requirements
       - bigdl-orca-spark3[ray] 
       - openvino==2022.3.0
       - protobuf==3.20.* (3.15.3 is **not acceptable**, 3.19.5 is ok)
       - pyarrow (version?)

### CODE REVIEW

File: .github/actions/orca/orca-python-horovod-tf2-py37-spark3-action/pr-test/action.yml [hunk 1]
```diff
@@ -16,6 +16,8 @@ runs:
         # install orca
         pip install -i https://pypi.org/simple python/orca/src/dist/bigdl_orca*-py3-none-manylinux1_x86_64.whl
```
Comment (hamham223) [inline .github/actions/orca/orca-python-horovod-tf2-py37-spark3-action/pr-test/action.yml:17]: No need to `pip uninstall bigdl-orca bigdl-orca-spark3` here. All the packages are removed in setting up env. 
e.g. https://github.com/intel-analytics/BigDL/blob/main/.github/actions/orca/setup-env/setup-horovod-tf2-py37/action.yml#L28

**Comment by hamham223** [comment]
@cyita Could you please take a look at the test result?

Job name: Orca-Python-Horovod-Tf2-Py37-Spark3
Link: https://github.com/intel-analytics/BigDL/actions/runs/4123606488/jobs/7137381928#step:6:7097
Results: 2 failed 3 passed

**Comment by hamham223** [comment]
https://github.com/intel-analytics/BigDL/pull/7475 This PR fixes the tutorial test issue.

**Review by cyita — APPROVED** [review]

File: python/orca/dev/test/run-pytests-spark-openvino.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+cd ../../
+
+echo "Running openvino test"
+python -m pytest -v test/bigdl/orca/learn/spark/test_estimator_openvino.py::TestEstimatorForOpenVINO::test_openvino_predict_ndarray
+python -m pytest -v test/bigdl/orca/learn/spark/test_estimator_openvino.py::TestEstimatorForOpenVINO::test_openvino_predict_ndarray_multi_input
+python -m pytest -v test/bigdl/orca/learn/spark/test_estimator_openvino.py::TestEstimatorForOpenVINO::test_openvino_predict_xshards
+# uncomment below pytests to run corresponding testing function
+# python -m pytest -v test/bigdl/orca/learn/spark/test_estimator_openvino.py::TestEstimatorForOpenVINO::test_openvino_multi_output
+# python -m pytest -v test/bigdl/orca/learn/spark/test_estimator_openvino.py::TestEstimatorForOpenVINO::test_openvino_predict_spark_df
```
Comment (hkvision) [inline python/orca/dev/test/run-pytests-spark-openvino.sh:34]: add some comments why these two tests can't run?

Comment (hkvision) [reply python/orca/dev/test/run-pytests-spark-openvino.sh:34]: @cyita ?

Comment (cyita) [reply python/orca/dev/test/run-pytests-spark-openvino.sh:34]: Will enable these two tests in another pr today.

**Pull request merged**